src/frontends/android/app/src/main/jni/Android.mk

```diff
@@ -1,4 +1,7 @@
 LOCAL_PATH := $(call my-dir)
+LOCAL_CFLAGS := -g
+LOCAL_STRIP_MODE := none
+LOCAL_STRIP_MODULE := keep_symbols
```

Comment:
Could you please describe why it is required?

Reply:
To generate debug symbols

Reply:
To keep them in generated libs